blurb/blurb.py

```diff
@@ -749,6 +749,10 @@ def find_editor():
         editor = os.environ.get(var)
         if editor is not None:
             return editor
+    editor = subprocess.check_output(
+        ["git", "config", "--get", "core.editor"]).strip()
+    if editor:
+        return editor
```

Comment:
Note, though, that the value from `git var GIT_EDITOR` cannot be blindly used on Windows since it defaults to just `vi`, which is not available on Windows outside of git bash or WSL. We should still check that the retrieved value can be found on PATH, and fall back to notepad if it can't.